zq2_spec.yaml

```
@@ -1,4 +1,4 @@
 versions:
-  - refspec: 033cc034a083151fded32092437302d042fe80ee
+  - refspec: d4c470e8fa69153b76a8233b6d26ae9a756e5da8
```

```suggestion
  - refspec: c4299949e26070914a7ecc5e70cee2f77baed385
```
Sorry I just made another change to the ZQ2 docs :)